app/splits/[id]/page.tsx

```diff
@@ -0,0 +1,124 @@
+'use client'
+
+import { Alert, AlertIcon, Box, Button, Container, Divider, Flex, VStack } from "@chakra-ui/react";
+import { useEffect, useState } from "react";
+
+import SplitGameList from "./split-game-list";
+import SplitInfo from "./split-info";
+
+async function getSplit(_id: string): Promise<Split> {
+  return {
+    splitChamp: "Mordekaiser",
+    splitTarget: 20,
+    id: "001",
+    splitGames: [
+      {
+        id: "1",
+        opponent: "Kayle",
+        opponentChampionId: "Kayle",
+        win: false,
+        earlyGamePlanning: {
+          winlvl1: false,
+          winlvl2: true,
+          winlvl3: true,
+          winlvl6: false,
+        }
+      },
+      {
+        id: "2",
+        opponent: "Chogath",
+        opponentChampionId: "Chogath",
+        win: true,
+        earlyGamePlanning: {
+          winlvl1: false,
+          winlvl2: true,
+          winlvl3: true,
+          winlvl6: false,
+        }
+      },
+      {
+        id: "3",
+        opponent: "Kayle",
+        opponentChampionId: "Chogath",
+        win: null,
+        earlyGamePlanning: {
+          winlvl1: null,
+          winlvl2: null,
+          winlvl3: null,
+          winlvl6: null,
+        }
+      },
+      {
+        id: "4",
+        opponent: "Illaoi",
+        opponentChampionId: "",
+        win: null,
+        earlyGamePlanning: {
+          winlvl1: null,
+          winlvl2: null,
+          winlvl3: null,
+          winlvl6: null,
+        }
+      },
+    ]
+  }
+}
+
+async function updateSplit(id: string, split: Split): Promise<Split> {
+  console.log("Trying to update split " + id + ".");
+
+  return split;
+}
+
+function ChampPracticeSplit({ params }: { params: { id: string } }) {
+  const [split, setSplit] = useState<Split>({ id: "", splitChamp: "", splitGames: [], splitTarget: 0 });
+  const [changesPending, setChangesPending] = useState(false);
```

Comment:
I suggest to look at useBoolean hook for these kind of state handlings https://usehooks-ts.com/react-hook/use-boolean
```suggestion
  const [changesPending, setChangesPending] = useState(false);
```

Reply:
I'll consider using that library.